lib/src/metrics/function_metric.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:meta/meta.dart';
+
+import '../models/metric_documentation.dart';
+import '../models/metric_value_level.dart';
+import '../models/scoped_class_declaration.dart';
+import '../models/scoped_function_declaration.dart';
+import 'metric.dart';
+
+/// Interface for metrics whose compute value for function node.
```

Comment:
Correction: `An interface for metrics that compute a value for a function node.`